cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -55,6 +55,18 @@ $receipt = @"
 "@
 $receipt_home = "${env:LOCALAPPDATA}\{{ app_name }}"
 
+if ($env:{{ disable_update_env_var }}) {
+  $install_updater = $false
+} else {
+  $install_updater = $true
+}
+
+$unmanaged_install = $env:{{ unmanaged_dir_env_var }}
+if ($unmanaged_install) {
+  $NoModifyPath = $true
```

Comment:
We should be deleting the -NoModifyPath `param`, people keep finding these weird inconsistent APIs we just shouldn't support them